ote/src/clj/ote/services/transport.clj

```diff
@@ -316,15 +306,14 @@
 
    (POST "/transport-operator/group" {user :user}
      (http/transit-response
-       (ensure-transport-operator-for-group db (-> user :groups first))))
+       (get-transport-operator db {::t-operator/ckan-group-id (get (-> user :groups first) :id)})))
```

Comment:
Can't we just remove this whole route? Is it needed anymore

Reply:
It is used in ckan-service-viewer